registry/handlers/app.go

```diff
@@ -249,6 +257,14 @@ func (app *App) dispatcher(dispatch dispatchFunc) http.Handler {
 			context.Repository = notifications.Listen(
 				repository,
 				app.eventBridge(context, r))
+
+			for _, mw := range app.Config.Middleware["repository"] {
+				rmw, err := repositorymiddleware.Get(mw.Name, mw.Options, context.Repository)
+				if err != nil {
+					panic(fmt.Sprintf("unable to configure repository middleware (%s): %s", mw.Name, err))
```

Comment:
Panicking in the middle of a request will be problematic. This should probably bail and log an error.